File: problems/0001-two-sum/solution.c
/**
 * Note: The returned array must be malloced, assume caller calls free().
 */
int* twoSum(int* nums, int numsSize, int target, int* returnSize){
    for (int x = 0; x < numsSize; x++) {
        for (int y = 1; y < numsSize; y++) {
            if ((nums[x]+nums[y] == target) && (x!=y)) {
                returnSize = malloc(8);
                returnSize[0] = x;
                returnSize[1] = y;
                return returnSize;
            }
        }
    }
    return returnSize;
}

File: problems/0001-two-sum/solution.py
class Solution(object):
    def twoSum(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        for i in range(len(nums)):
            nums2 = nums[i+1:]
            for j in range(len(nums2)):
                if nums[i] + nums2[j] == target:
                    return([i,i+j+1])